index.html

```diff
@@ -0,0 +1,195 @@
+              of sharing and opening.
+            </p>
+          </div>
+
+          <div class="program-cards">
+            <img class="card-images" src="images/program_icon_02.png" />
+            <p class="card-title">CC Exhibition</p>
+            <p class="card-text">
+              Appreciate various creations applying CC license of artists,
+              organized from Art Center Nabi
+            </p>
+          </div>
+
+          <div class="program-cards">
+            <img class="card-images" src="images/program_icon_03.png" />
+            <p class="card-title">Forum</p>
+            <p class="card-text">
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+          <div class="program-cards">
+            <img class="card-images" src="images/program_icon_04.png" />
+            <p class="card-title">Workshop</p>
+            <p class="card-text">
+              Try creating your own work using open source license rather than
+              just watching at it.
+            </p>
+          </div>
+          <div class="program-cards">
+            <img class="card-images" src="images/program_icon_05.png" />
+            <p class="card-title">CC Ignite</p>
+            <p class="card-text">
+              get opportunities to network with programmers around the world,
+              also after the conference.
+            </p>
+          </div>
+        </div>
+        <a href="#" class="join-link"> Join 2023 Web Developer Conference</a>
+        <a href="#" class="whole-program"> SEE THE WHOLE PROGRAM</a>
+      </section>
+      <!-- end of main program section -->
+      <!-- featured speakers section -->
+      <section class="speakers-wrapper" id="speakers">
+        <h2 class="speakers-title">Featured Speakers</h2>
+        <hr class="title-underline" />
+        <div class="speakers">
+
+        </div>
+      </section>
+      <!--  end of featured speakers section -->
+      <!--  partner section -->
+      <section class="partner-wrapper" id="partner">
+        <h2 class="partner-title">Our Partners</h2>
+        <hr class="title-underline" />
+        <div class="partner-logo">
+          <a> <img src="images/mozilla.png" alt="Mozilla logo here" /></a>
+          <a> <img src="images/googlekorea.png" alt="Google logo here" /></a>
+          <a> <img src="images/naver.png" alt="Naver logo here" /></a>
+          <a>
+            <img src="images/daumkakaologo.png" alt="daumkakao logo here"
+          /></a>
+          <a> <img src="images/airbnb.png" alt="Airbnb logo here" /></a>
+        </div>
+      </section>
+      <!--  end of partner section -->
+    </main>
+    <footer>
+      <a href="#"
+      ><img class="footer-logo" src="images/eta_logo.png" alt="ETA institution logo here"
+    /></a>
+      <p class="copyright">&copy; Eagle Tech Academy. Some Rights Reserved.</p>
+    </footer>
```

Comment:
- [ ] Kindly consider matching the design guidelines by making the footer content in the center and giving space evenly between elements as shown below

| Your design  | Design guidline |
| ------------- | ------------- |
| ![image](https://github.com/mulugithub/module-one-capstone/assets/53402600/7349a656-0476-4b7d-95f8-3e8a9b6eee34)  | ![image](https://github.com/mulugithub/module-one-capstone/assets/53402600/83b56d51-f070-479d-b57f-fdb77b44647b) |

- [ ] Also consider the footer to have the same background as exist in the design guideline
  - In Desktop
       - About Page: It has a dark background.
       - Home Page: It has a white background.